atlast_sc/data.py

```diff
@@ -164,7 +164,7 @@ def __post_init__(self):
 
     # Forward Efficiency
```

Comment:
Could we add a reference here?
```suggestion
    # Forward Efficiency - 0.95 based on ALMA Memo 602(https://library.nrao.edu/public/memos/alma/main/memo602.pdf), page 8
```